docs/source/usersguide/input.rst

```diff
@@ -558,19 +559,127 @@ The following code snippet illustrates the instantiation of the ``CPUSolver`` fo
+  solver.setConvergenceThreshold(tolerance)
+
+  # Set the flux in the source cell to unity and compute the flux
+  solver.setFixedSourceByCell(source_cell, 1, 1.0)
+  solver.computeFlux(max_iters)
+
+The first group flux is plotted below. All other flux plots are zero throughout the entire geometry because the scattering source is not accounted in ``computeFlux(...)`` and neutrons are always born in the first group. By using the ``computeFlux(...)`` routine, OpenMOC is directed to only use the provided fixed source and not to update for fission or scattering. Notice that there are prominant ray effects since there is no scattering in this problem, an inherent characteristic of MOC solvers. The MOC solver is defined with 4 azimuthal angles for the figure on the left and 16 azimuthal angles for the figure on the right. As the number of angles increases, the effect is not as noticable but is still present.
+
+.. _figure_fixed_source_flux_calc:
+     
+.. table:: 
+   
+   +--------------------------------------------------------+--------------------------------------------------------+
+   | .. _figa:                                              | .. _figb:                                              |
+   |                                                        |                                                        |
+   | .. image:: ../../img/fs-flux-calc-4-angles-group-1.png | .. image:: ../../img/fs-flux-calc-32-angles-group-1.png|
+   |   :width: 50 %                                         |   :width: 50 %                                         |
+   |   :align: right                                        |   :align: left                                         |
+   +--------------------------------------------------------+--------------------------------------------------------+
+
+While this case seems illsuited for the ``computeFlux(...)`` routine, the ``computeFlux(...)`` would be very useful for the fixed source cosine distribution problem described previously above. This problem
+
+TODO: make fixed source plots
+
+Source Calculations
+-------------------
+
+In other problems, the source distribution is desired for a set eigenvalue. For this case, the ``computeSource(...)`` routine can be used, which calculates the total source (including any fixed source) in each flat source region iteratively. At the end of each transport sweep, the eigenvalue is reset to the eigenvalue set by the user. By default this value is 1.0 if no value is provided by the user. Returning to the first problem discussed using the ``computeFlux(...)`` routine where a fixed source was placed in a geometry filled with water, the ``computeFlux(...)`` routine could not account for scattering. Since the ``computeSource(...)`` routine computes both scattering and fission sources during transport sweeps, it is able to account for scattering and compute the correct flux distribution. The line below shows how the ``computeSource(...)`` routine can be called.
```

Comment:
Have `if no value is provided by the user` sounds a little verbose and I think can be removed.